modules/menu/menu.c

```diff
@@ -941,6 +941,8 @@ static void ua_event_handler(struct ua *ua, enum ua_event ev,
 	(void)prm;
 	(void)arg;
 
+	conf_get_bool(conf_cur(), "ringback_disabled", &menu.ringback_disabled);
+
```

Comment:
can you please move this to `module_init` ?

it is enough to read this config value once, during init.

Reply:
will do.